src/Controls/src/Core/PlatformConfiguration/AndroidSpecific/ImageButton.cs

```diff
@@ -120,5 +120,36 @@ public static IPlatformElementConfiguration<Android, FormsImageButton> SetShadow
 			return config;
 		}
 		#endregion
+
+		#region RippleColor
+		
+		/// <summary>Bindable property for attached property <c>RippleColor</c>.</summary>
+		internal static readonly BindableProperty RippleColorProperty = BindableProperty.Create("RippleColor", typeof(Color), typeof(ImageButton), default);
+
+		/// <include file="../../../../docs/Microsoft.Maui.Controls.PlatformConfiguration.AndroidSpecific/ImageButton.xml" path="//Member[@MemberName='GetRippleColor']/Docs/*" />
+		public static Color GetRippleColor(BindableObject element)
```

Comment:
Make this internal for net9, and then open a new issue for us to make public in net10 or if the powers that be allow us to add ABI.